packages/compose/src/hooks/use-media-query/index.js

```diff
@@ -1,7 +1,25 @@
 /**
  * WordPress dependencies
  */
-import { useState, useEffect } from '@wordpress/element';
+import { useMemo, useSyncExternalStore } from '@wordpress/element';
+
+/**
+ * A new MediaQueryList object for the media query
+ *
+ * @param {string} [query] Media Query.
+ * @return {MediaQueryList|null} A new object for the media query
+ */
+function getMediaQueryList( query ) {
+	if (
+		!! query &&
```

Comment:
Style nit: you can remove the `!!`. That's useful only when you're returning/passing the result of some boolean expression and want to make sure it's really a boolean.

Reply:
Good point.